lib/EndOfItem/tests/EndOfItem.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import '../../../test/jest/__mock__';
+
+import { renderWithIntl } from '../../../test/jest/helpers';
+import { EndOfItem } from '../EndOfItem';
+
+const setupEndOfItem = ({
+  title,
+  className,
+} = {}) => renderWithIntl(
+  <EndOfItem
+    title={title}
+    className={className}
+  />,
+  [{
+    prefix: 'stripes-components',
+    translations: { endOfList: 'stripes-components.endOfList' },
+  }]
+);
+
+describe('EndOfItem', () => {
+  const customClass = 'my-class';
+
+  describe('rendering EndOfItem with title', () => {
+    const title = 'Title';
+
+    it('should add custom class style to layout container', () => {
+      setupEndOfItem({
+        title,
+        className: customClass,
+      });
+      const layout = document.querySelector('[data-test-layout]');
+
+      expect(layout.classList.contains(customClass)).toBeTruthy();
+    });
+
+    it('should display "end mark" svg icon', () => {
+      setupEndOfItem({ title });
+      const icon = document.querySelector('[data-test-icon-element]');
+      const hasIconEndMarkClass = icon.classList.contains('icon-end-mark');
+
+      expect(icon).toBeInTheDocument();
+      expect(hasIconEndMarkClass).toBeTruthy();
+    });
+
+    it('should display passed title', () => {
+      setupEndOfItem({ title });
+      const label = document.querySelector('span.label');
+
+      expect(label.innerHTML).toEqual(title);
+    });
+  });
+
+  describe('rendering EndOfItem without properties', () => {
+    it('should not add custom class style to layout container', () => {
+      setupEndOfItem();
+      const layout = document.querySelector('[data-test-layout]');
+
+      expect(layout.classList.contains(customClass)).toBeFalsy();
+    });
+
+    it('should display default title', () => {
+      setupEndOfItem();
+      const label = document.querySelector('span.label');
+
+      expect(label.innerHTML).toEqual('stripes-components.endOfList');
```

Comment:
Shouldn't it be translation rather than translation key?

Reply:
In this case, it is okay. In the components project, we don't use translations from stripes-components.

Reply:
Then why use translation at all?

Reply:
Agreed to change the string to omit the info that it is taken from stripes-components.